_topic_maps/_topic_map_rosa.yml

```diff
@@ -88,6 +88,8 @@ Topics:
   File: rosa-mobb-configure-custom-tls-ciphers
 - Name: Verifying Permissions for a ROSA STS Deployment
   File: rosa-mobb-verify-permissions-sts-deployment
+- Name: Configuring ROSA to use Azure AD group claims
```

Comment:
Configuring ROSA to use Microsoft Entra ID (formerly Azure Active Directory) group claims